src/lucky/redirectable.cr

```diff
@@ -42,29 +42,29 @@ module Lucky::Redirectable
     redirect to: action.route, status: status
   end
 
+  # Redirect using a `String` and a `Status` value
+  #
+  # ```crystal
+  # redirect to: "/users", status: Status::MovedPermanently
+  # ```
+  # You can find a list of all of the possible statuses [here](https://github.com/luckyframework/lucky/blob/master/src/lucky/action.cr).
+  def redirect(to path : String, status : Lucky::Action::Status = Lucky::Action::Status::Found) : Lucky::TextResponse
```

Comment:
I think we can get rid of the default here since the other one already has a default. What do you think? Does that still work?

Reply:
Yup. It still works.